extension/lib/tracker.js

```diff
@@ -99,6 +99,10 @@ class Tracker {
         return $("#" + this.readableDomIds[containerId]);
     }
 
+    getFirstContainer() { // Added to access container ID
```

Comment:
ctrl-f ing around, is this used anywhere?

Reply:
Nope; just nuked it